.gitignore

```diff
@@ -1,3 +1,3 @@
-node_modules/
-package-lock.json
```

Comment:
It's actually recommended to commit the package-lock.json for continuous integration.

One day NPM will allow reproducible builds. They are not quite there yet :)